package.json

```diff
@@ -26,7 +26,9 @@
     "@actions/github": "^1.1.0",
     "conventional-changelog-conventionalcommits": "4.4.0",
     "conventional-commit-types": "^3.0.0",
-    "conventional-commits-parser": "^3.1.0"
+    "conventional-commits-parser": "^3.1.0",
+    "probot": "^7.5.3",
+    "probot-config": "^1.0.0"
```

Comment:
It would be great to stick to Github native tooling and avoid Probot.

Instead, you should be able to pass arguments to the Github action.

See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith